lib/spark_api/models/finders.rb

```diff
@@ -6,6 +6,7 @@ module Finders
 
       def find(*arguments)
         scope = arguments.slice!(0)
+        raise ArgumentError, "argument can't be nil" if scope.nil?
```

Comment:
It seems like this would read better rolled in to the `case` statement below. The main advantage I see to putting this here is avoiding the overhead of setting up the `options`. In this case, I would favor readability, as the overhead seems low. Am I overlooking other advantages?